### DISCUSSION TITLE
Printing Copyright page

### DISCUSSION THREAD

**Discussion by drfridline** [fmarotta/kaobook#263] [General]
Trying to compile the example and everything works fine except I can't get the copyright page to print.  Has anyone else experienced this issue?  I am running MacTeX 2022.

**Comment by omasanori** [comment]
I confirmed the same behavior with pdfLaTeX and LuaLaTeX (LuaHBTeX) included in TeX Live 2021 packaged as Ubuntu 22.04's APT packages.

**Comment by omasanori** [comment]
OK, I figured out the cause. From the KOMA-Script's manual:

> In two-sided printing, the standard classes leave the back (verso) of the title page empty. However, with KOMA-Script the back of the full title page can be used for other information. There are exactly two elements which the user can freely format: titlebackhead and titlebackfoot. The header can extend to the footer and vice versa. Using this guide as an example, the legal disclaimer was set with the help of the \uppertitleback command.

So, `twoside=false`, as in the `minimal_book` example, disables `\uppertitleback` (and `\lowertitleback`). Use `twoside=true` and you will get the copyright page.
